### ISSUE TITLE
DropDown: Keyboard focus is not visible when High Contrast mode is enabled.

### ISSUE THREAD

**Issue by chshrikh** [OfficeDev/office-ui-fabric-react#5238]
**Test Environment Details:**
OS: Windows 10 RS3
OS version: build 16299.371
Browser: Firefox
Firefox version: 60.0(32-Bit)
Version: v/20180515T155752

**Steps to reproduce:**
1. Open https://developer.microsoft.com/en-us/fabric#/components/dropdown
2. Navigate to different drop downs with keys

**Expected Behavior:**
Active keyboard focus should be highlighted by dotted lines/Solid lines.
While navigating keyboard focus should be clearly visible on drop down boxes

**Actual Behavior:**
While navigating keyboard focus is not visible on drop down boxes.

**Comment by dzearing** [comment]
@lynamemi Is it possible you could help with this?

**Comment by natalieethell** [comment]
@cliffkoh I know this issue is a little older (and perhaps the original issue was fixed?) but it looks like in your PR here https://github.com/OfficeDev/office-ui-fabric-react/pull/5661 you changed 

```
@include high-contrast {
    background-color: Highlight;
    border-color: Highlight;
    color: HighlightText;
  }

```

to 
```
[HighContrastSelector]: {
      backgroundColor: 'Highlight',
      borderColor: 'Highlight',
      color: 'Highlight'
    },
```

So now, the text is no longer visible when selected in HC. 

See below:
![image](https://user-images.githubusercontent.com/14134000/43980027-79e10658-9ca1-11e8-99a9-9fd7892d1849.png)


It looks like changing color to `HighlightText` fixes the issue.

**Comment by micahgodbolt** [comment]
Still a bug, should be a quick fix changing back to HighlightText and testing

![image](https://user-images.githubusercontent.com/1434956/44685001-47228780-a9ff-11e8-82ee-1aea7afbac6b.png)
